packages/runner/src/runner.ts

```diff
@@ -371,14 +371,16 @@ export class Runner<X extends AppConfig> implements IComponent {
         try {
             await this.runSequence(sequence, args);
 
+            this.writeMonitoringMessage([RunnerMessageCode.SEQUENCE_COMPLETED, {}]);
             this.logger.trace(`Sequence completed. Waiting ${exitDelay}ms with exit.`);
 
-            await defer(exitDelay); // @TODO: investigate why we need to wait
+            await defer(exitDelay);
```

Comment:
Do you know why the delay is needed?

Reply:
yes, it is needed to perform API checks after sequence finish

Reply:
mainly for our BDD tests